FlowVolume/CS/README.md

```diff
@@ -0,0 +1,111 @@
+---
+page_type: sample
+urlFragment: flow-volume
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise
+description: This sample demonstrates how to use a relatively inexpensive device and a UP Board to measure the volume of liquid flowing through a hose.
+---
+
+# Flow Volume
+
+In this sample, we will demonstrate how to use a relatively inexpensive device and a UP Board to measure the volume of liquid flowing through a hose.
+
+Keep in mind that the GPIO APIs are only available on Windows 10 IoT Enterprise, so this sample cannot run on your desktop.
+
+## Step 1: Set up your hardware
+___
+The hardware setup for this sample is relatively simple.
+
+You’ll need a few components:
+
+*   a Hall-effect flow sensor (we used a FL-S402B flow meter by DIGITEN)
+
+![Flow Sensor](../../Resources/FlowSensor.jpg)
+
+*   a microcontroller board running Windows IoT Enterprise (we used a UP Board) with a free GPIO pin
+
+![Assembled Components](../../Resources/Assembled.jpg)
+
+### For UP Board
+
+1.  Connect the red wire to 3.3V (pin 1 on the expansion header) on the UP Board.
+2.  Connect the black wire to ground (pin 9).
+3.  Connect the yellow wire, the data pin, to the GPIO-2 pin that we will read from (we used pin 7 for convenience, since it is right there between pins 1 and 9).
+4.  Connect the monitor to your device (we used an HDMI cable to connect our Upbaord to a standard monitor).
+
+For reference, here is the pinout of the UP Board:
+
+![](../../Resources/UP Board_Pinout.png)

```

Comment:
Please fix this filepath so the image can populate

Reply:
Requested to maintain the "upboard" in the image file path, If  you are changing it to "UP Board" in the image path. the image is not coming up. It might be due to Space in "UP Board". 

I am Changing image file path "UP Board" back to "upboard".